### PR TITLE
This change creates the hability to order jobs by their last executio…

### PR DESCRIPTION
**Pull request by Dudssource**
Fixing the issue #113 , creating the hability to order jobs by their last execution.

### CODE REVIEW

File: src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/order/ByLastBuild.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * Inspired in <a href="https://github.com/Mercynary">@Mercenary</a>'s answer to
+ * issue <a href="https://github.com/jan-molak/jenkins-build-monitor-plugin/issues/113">#113</a>.
+ */
+public class ByLastBuild implements Comparator<Job<?, ?>> {
+
+    @Override
+    public int compare(Job<?, ?> a, Job<?, ?> b) {
+
+    	// none of then executed
+    	if (a.getLastBuild() == null && b.getLastBuild() == null) {
```
Comment (jan-molak) [inline src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/order/ByLastBuild.java]: Just to confirm, do you mean `getLastBuild` or `getLastCompletedBuild`? If we use `getLastBuild` the job will be moved to an earlier position in the list even if it hasn't finished running. Is that what you wanted?

**Comment by stein321** [comment]
Our team would love to use this feature. Is there anything holding it up from being merged?

**Comment by jan-molak** [comment]
Hey @stein321 and thanks for your feedback - I'm not sure if @Dudssource (and yourself) would like to order by 'last build' (which includes builds that are still running) or by 'last completed build', hence my comment on the PR.

**Comment by stein321** [comment]
Hey @jan-molak , order by last build including current running builds would be stellar.

**Comment by ravikalaga** [comment]
Hey @jan-molak I work with Ben; any chance you can complete the merge so that we can use it?

**Comment by Dudssource** [comment]
Hi, I'm sorry for taking too long to answer that @jan-molak ! I believe that including the current running builds would be great.

**Comment by ravikalaga** [comment]
@jan-molak Any luck with merging this? Thanks!

**Comment by jan-molak** [comment]
Hey guys and sorry for keeping you waiting :-) If you'd like to take the latest build for a spin before it hits the Jenkins Update Center you can [download it here](https://smartcode-opensource.ci.cloudbees.com/job/build-monitor/34/artifact/build-monitor-plugin/target/build-monitor-plugin.hpi).

Looking forward to hearing your thoughts!
Jan

**Comment by ravikalaga** [comment]
Thanks Jan. We will try to get it running and get back to you. 

Ravi.

**Pull request merged**